controllers/batterycontroller.py

```diff
@@ -1,16 +1,6 @@
-from TestData import*
-from canBus import*
-from Controller import*
-class BatteryController(Controller):
+class BatteryController():
+  testData
+  CANBusController
   def __init__(self):
-     self.testData = TestData()
-     self.testData.loadData()
-     self.CANBusController = canBus() #uhm not sure what or how to initialize this
-
-     registerAction("shutdown", shutdown)
-     registerAction("update", update)
-
-     #should the batterycontroller have a voltage variable?
-  def shutdown(self):
-
-  def update(self):
+     self. testData.loadData()
+     self.CANBusController = new canBus() #uhm not sure what or how to initialize this
```

Comment:
For line 6, provided the relevant library will contain a class called canBus() (preferably named CANBus()), you add the appropriate initialization arguments, and you import it as the top, this should work fine.